package.json

```diff
@@ -169,6 +169,8 @@
     "react-test-renderer": "^16.13.1",
     "rollup": "^2.33.2",
     "rtlcss-webpack-plugin": "^4.0.4",
+    "sass": "^1.30.0",
```

Comment:
Agreed. Let's do this without Sass for now.

If all we want here is nesting and variables, I'd use PostCSS and `postcss-loader`

Reply:
I was not aware about not using scss, so added it thinking it will help in case we need. I did read about the issue with node-sass, so didn't use that but this uses sass based on dart, which is also recommended by webpack.

As you mentioned, since it's been decided not to use it, I'll update these to use postcss. Thank you for the suggestion @swissspidy

Reply:
In the future, if you are going to add a library and change the build process like, this, I would strongly recommend flagging this first. Also a change of this size, should be documented  and explained in the PR description.